.devcontainer/post-create.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Add welcome message
+sudo cp .devcontainer/welcome.txt /usr/local/etc/vscode-dev-containers/first-run-notice.txt
+
+# Switch to vets-api ruby version
+asdf install ruby $( cat .ruby-version )
+asdf global ruby $( cat .ruby-version )
+
+git clone https://github.com/department-of-veterans-affairs/vets-api-mockdata.git ../vets-api-mockdata
+
+sudo apt update
+sudo apt install -y libpq-dev pdftk shared-mime-info postgresql-11-postgis-3
```

Comment:
We're on pg15 now

Reply:
thanks! I was going off the docker files, I'll update this and the native setup README.